packages/blade/docs/guides/Installation.stories.mdx

```diff
@@ -40,47 +40,62 @@ Before you install the package, make sure that you have performed the following
 
 ## ⬇ Add blade to your application
 
-1. Install blade as a dependency. 
-Blade has a peer dependency on a few libraries, you can skip adding it if you already have it installed in your project.
+1. Install blade as a dependency.
+   Blade has a peer dependency on a few libraries, you can skip adding it if you already have it installed in your project.
 
-  - `styled-components`
+- `styled-components`
   > **Note**
   >
   > Currently, blade only supports styled-components v5 only
 
-   ```shell
-   yarn add @razorpay/blade styled-components@5.3.11
-   ```
-2. Follow [this guide](#-install-fonts) to install the fonts.
+```shell
+yarn add @razorpay/blade styled-components@5.3.11
```

Comment:
move this shell block before styled components and put all the dependencies including i18nify in one shell block itself as we have done in React native section below and then you can use bullt points to describe all the dependencies together

Reply:
Done
![Uploading Screenshot 2024-02-12 at 3.02.11 PM.png…]()